static/js/event-display.js

```diff
@@ -1,4 +1,9 @@
-const urlVars = getUrlParams();
+const urlParams = getUrlParams();
```

Comment:
use destructuring?

Reply:
There is some parsing involved for some of the parameters, so I prefer extracting all the params on their own line.